frontend/src/components/Footer.tsx

```diff
@@ -75,23 +74,26 @@ const Footer = () => {
     <USWDSFooter
       data-testid="footer"
       size="medium"
-      primary={<div />}
+      returnToTop={
+        <div className="grid-container usa-footer__return-to-top">
+          <a href="#">{t("return_to_top")}</a>
+        </div>
+      }
```

Comment:
This maybe got lost in the shuffle. I don't think I included it in the mockup, so it might've been nixed on purpose. It's recommended by USWDS guidance. Added it back, but I could go either way. I don't find it all that useful.